src/pages/CirclesPage/getOrgData.ts

```diff
@@ -0,0 +1,37 @@
+import { order_by } from 'lib/gql/__generated__/zeus';
```

Comment:
hmm... i dunno, this query is really specific to the needs of this page. i feel like it's misleading to move it somewhere more general, because it suggests it should be reused. i liked the idea of organizing files by feature that someone posted in chat the other day (maybe it was @META-DREAMER) and this is in line with that.

Reply:
"ease of reference" is an important goal, but i think we get that well enough by searching for uses of client.query, e.g.:

![Screen Shot 2022-04-01 at 12 47 09 PM](https://user-images.githubusercontent.com/193667/161331635-b057d696-521a-4b20-ba33-feab72f283ed.png)

Reply:
Makes sense.